module-2/backend/week_one.md

```diff
@@ -5,28 +5,116 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+
+  *GET - read data
+  POST - insert data
+  PUT - update data
+  PATCH - update data?
+  DELETE - delete data*
+
+  
 2. What is Sinatra?
-4. What is MVC?
-5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
-6. What types of variables are accessible in our view templates without explicitly passing them?
-7. Given the following block of code, how would I pass an instance variable `count` with a value of `1` to my `index.erb` template?
+
+ *Sinatra is a gem you install that makes it easy to test and host new web applications locally*

```

Comment:
we can also host our sinatra applications on a third party server - aka AWS or Heroku so they aren't just available locally!